### PR TITLE
Updated dev-tools

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Fixes all the warnings here: https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/657490890.

### CODE REVIEW

File: dev-tools/phive.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phive xmlns="https://phar.io/phive">
-  <phar name="composer-normalize" version="^2.1.2" installed="2.1.2" location="./tools/composer-normalize" copy="false"/>
-  <phar name="composer-require-checker" version="^2.0" installed="2.0.0" location="./tools/composer-require-checker" copy="false"/>
-  <phar name="phpstan" version="^0.12" installed="0.12.9" location="./tools/phpstan" copy="true"/>
+  <phar name="composer-normalize" version="^2.2" installed="2.2.4" location="./tools/composer-normalize" copy="false"/>
+  <phar name="composer-require-checker" version="^2.1" installed="2.1.0" location="./tools/composer-require-checker" copy="false"/>
+  <phar name="phpstan" version="^0.12" installed="0.12.11" location="./tools/phpstan" copy="true"/>
```
Comment (keradus) [inline dev-tools/phive.xml]: can we somehow drop `installed` lock?

Comment (julienfalque) [reply dev-tools/phive.xml]: We should not: a new patch version of PHPStan can report new errors. Not locking the version would make our CI fail on a regular basis.

Comment (GrahamCampbell) [reply dev-tools/phive.xml]: 0.12.12+ totally crashes, actually.

**Comment by SpacePossum** [comment]
Can you update the travis script to output the versions of the versions of the tooling used (the changed ones), as I think we currently do not see this? After that, RTM from me

**Comment by GrahamCampbell** [comment]
The versions are already shown?

**Comment by SpacePossum** [comment]
all I see is
```
0.18s$ ./dev-tools/tools/composer-normalize composer.json --dry-run
composer.json is already normalized.
```
https://travis-ci.org/github/FriendsOfPHP/PHP-CS-Fixer/jobs/662693195?utm_medium=notification&utm_source=github_status

maybe I missing something?

**Comment by GrahamCampbell** [comment]
> Linking /home/travis/.phive/phars/composer-normalize-2.3.2.phar to ./tools/composer-normalize

**Comment by GrahamCampbell** [comment]
Full version:

```
λλλ phive packages

Phive 0.13.2 - Copyright (C) 2015-2020 by Arne Blankerts, Sebastian Heuer and Contributors

Downloading https://api.github.com/repos/ergebnis/composer-normalize/releases

Downloading https://github.com/ergebnis/composer-normalize/releases/download/2.3.2/composer-normalize.phar

Downloading https://github.com/ergebnis/composer-normalize/releases/download/2.3.2/composer-normalize.phar.asc

Linking /home/travis/.phive/phars/composer-normalize-2.3.2.phar to ./tools/composer-normalize

Downloading https://api.github.com/repos/maglnet/ComposerRequireChecker/releases

Downloading https://github.com/maglnet/ComposerRequireChecker/releases/download/2.1.0/composer-require-checker.phar

Downloading https://github.com/maglnet/ComposerRequireChecker/releases/download/2.1.0/composer-require-checker.phar.asc

Linking /home/travis/.phive/phars/composer-require-checker-2.1.0.phar to ./tools/composer-require-checker

Downloading https://api.github.com/repos/phpstan/phpstan/releases

Downloading https://github.com/phpstan/phpstan/releases/download/0.12.11/phpstan.phar

Downloading https://github.com/phpstan/phpstan/releases/download/0.12.11/phpstan.phar.asc

Copying phpstan-0.12.11.phar to ./tools/phpstan
```

**Comment by SpacePossum** [comment]
>Can you update the travis script to output the versions of the versions of the tooling use

I'm interested in the versions of the tools are _used_ by Travis, I did see which are downloaded

**Pull request closed without merging**